openqabot/__init__.py

```diff
@@ -1,16 +1,24 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: MIT
+
+import os
+
 # Used configuration parameters, e.g. api url's
-QEM_DASHBOARD = "http://dashboard.qam.suse.de/"
-SMELT = "https://smelt.suse.de/graphql"
-OBS_URL = "https://api.suse.de"
+QEM_DASHBOARD = os.environ.get("QEM_DASHBOARD_URL", "http://dashboard.qam.suse.de/")
+SMELT_URL = os.environ.get("SMELT_URL", "https://smelt.suse.de/graphql")
```

Comment:
```diff
-SMELT_URL = os.environ.get("SMELT_URL", "https://smelt.suse.de/graphql")
+SMELT_URL = os.environ.get("SMELT_URL", "https://smelt.suse.de")
```

Reply:
@baierjan `"https://smelt.suse.de/graphql"` should be correct endpoint, if there isnt any code change which adds `graphql` later